src/test/java/ru/training/at/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,99 @@
+package ru.training.at.hw2.ex1;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class Exercise1 {
+    private WebDriver webDriver;
+
+    @BeforeTest
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver",
```

Comment:
Because of that line I can't run your project locally without additional actions. You could put Driver in project resources or pay attention to WebDriverManager library from bonigarcia - it helps with driver initializing in project without any problems.